src/KafkaW/Message.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include <cstdlib>
+#include <flatbuffers/flatbuffers.h>
+
+namespace KafkaW {
+
+class Message {
+public:
+  explicit Message(flatbuffers::DetachedBuffer InputBuffer)
+      : Buffer(std::move(InputBuffer)) {}
+
+  char *data() { return reinterpret_cast<char *>(Buffer.data()); }
+  size_t size() { return Buffer.size(); }
```

Comment:
The methods of `Message` are not covered by unit tests. Perhaps worth adding a test where you serialise a simple message, put it in a Message, and then check you can get it back out and deserialise it?